pkg/apis/openebs.io/v1alpha1/blockdeviceclaimbuilder.go

```diff
@@ -144,6 +148,23 @@ func (bdc *BlockDeviceClaim) WithBlockVolumeMode(mode corev1.PersistentVolumeMod
 	return bdc
 }
 
+// WithBlockDeviceTag appends (or creates) the BDC Label Selector
+// by setting the provided value to the fixed key
+// openebs.io/block-device-tag
+// This will enable the NDM to pick only devices that
+// match the node (hostname) and block device tag value.
+func (bdc *BlockDeviceClaim) WithBlockDeviceTag(bdTagValue string) *BlockDeviceClaim  {
+	if bdc.Spec.Selector == nil {
+		bdc.Spec.Selector = &metav1.LabelSelector{}
+	}
+	if bdc.Spec.Selector.MatchLabels == nil {
+		bdc.Spec.Selector.MatchLabels = map[string]string{}
```

Comment:
@kmova one question why do you prefer `map[string]string{}` than `make(map[string]string)` ?

Reply:
East west programming! I saw that being used in the file. 

Is `make(map[string]string)` the recommended way?

Reply:
Both are same and free to choose. 
Only diff is make allows you to specify capacity while initialising which is trivial
and not required in case of maps as it serves no purpose.

Reply:
Although people agreeing on a consistent code style for a repo can impose to choose one.

Reply:
@kmova  I am good with the changes.